index.page.tsx

```diff
@@ -1,10 +1,9 @@
-import * as React from 'react';
 import { useTranslate } from '@portal/hooks';
 
 export const frontmatter = {
   seo: {
-    title: 'Home | XRPL.org',
-    description: "XRPL.org is a community-driven site for the XRP Ledger (XRPL), an open-source, public blockchain. Get technical documentation, reference materials, and tools.",
+    title: 'Crypto Currency Ledger Blockchain Company | XRPL.org',
```

Comment:
xrpl.org is the XRP Ledger developer portal and hosts documentation, blog, and other developer resources. 
```suggestion
    title: 'XRP Ledger Home | XRPL.org',
```